compliance/collection/metrics/metrics.go

```diff
@@ -0,0 +1,53 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/metrics"
+)
+
+var (
+	numberOfRHELPackages = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
I chose a Gauge, as it was described as a value that can raise and lower, as opposed to a Counter, which is monotonically increasing. The number of packages is something that might fall. 

On the other hand, we don't really expect the package count on the same node to ever fall. It might increase though, especially with layering in mind.

Reply:
Okay, let's keep it as gauge.